ecobee/auth.go

```diff
@@ -206,6 +208,17 @@ type Client struct {
 // (https://www.ecobee.com/consumerportal/index.html#/dev)
 func NewClient(clientID, authCache string) *Client {
 	return &Client{oauth2.NewClient(
-		context.Background(),
-		TokenSource(clientID, authCache))}
+		context.Background(), TokenSource(clientID, authCache))}
+}
+
+// GetPin retrieves an ecobee Pin and Code, allowing calling code to present them to the user
+// outside of the ecobee request context.
```

Comment:
Maybe add a sentence about how this is useful when you need to split Authorization from creation of the access token itself?

Reply:
👍

Reply:
fixed